DEVELOPING.md

```diff
@@ -0,0 +1,60 @@
+# Developing Klaxon Locally #
+
+If you're interested in [helping improve Klaxon by contributing code](CONTRIBUTING.md), this is a quick guide to how to get it running on your local machine for development.
+
+First, we'll assume you already have git on your local machine, as well as a standard postgres installation. So go to your terminal, navigate to whatever directory where you keep your projects and clone the Klaxon repo.
+
+```
+git clone git@github.com:themarshallproject/klaxon.git
+```
+
+After you've cloned it, `cd` into the Klaxon directory. If you're on a Mac, you'll need to already have [homebrew](https://brew.sh/) and [rbenv](https://github.com/rbenv/rbenv) (a program that manages versions of Ruby) installed. Then you'll want 
+
+These next commands will put the proper (albeit aged) version of Ruby that Klaxon requires on your machine and make it available in this repo's directory.
+
+```
+brew update && brew upgrade ruby-build
+rbenv install
+``` 
+
+Next, you'll need to have the proper versions of all of Klaxon's dependency libraries, as well as foreman, Heroku's tool for running a local server for development. Run these commands here.
```

Comment:
Minor detail, but foreman is not made by Heroku and is more general than this description would indicate. I would describe it as a "process manager".

Reply:
Ahhh. That's my mistake. I was introduced to it through Heroku and always assumed they had created foreman.